client/src/components/SudoModePasswordField/SudoModePasswordField.js

```diff
@@ -15,14 +15,50 @@ function SudoModePasswordField(props) {
   const {
     onSuccess,
     autocomplete,
+    initiallyCollapsed,
     verifyMessage,
+    sectionTitle,
   } = props;
   const passwordFieldRef = createRef();
   const [responseMessage, setResponseMessage] = useState('');
   const [showVerify, setShowVerify] = useState(false);
+  // setting to null initially to prevent a fout as useEffect()
+  // below will cause a re-render when initiallyCollapsed
+  const [expanded, setExpanded] = useState(null);
+
+  useEffect(() => {
+    setExpanded(!initiallyCollapsed);
+  }, [initiallyCollapsed]);
```

Comment:
As per https://github.com/silverstripe/silverstripe-admin/pull/1912#pullrequestreview-2676837781 I don't think this additional state is necessary.

This PR effectively moved the "verify" button from below the text to be next to it, right? (There are other changes like adding `sectionTitle` but they aren't affected by this state).
The button does the same thing in both cases - hides the "verify" button and displays a text field and a submit button.

I don't understand why this state is needed for that given it's the same functionality in both cases.

Reply:
Have removed expanded state